### DISCUSSION TITLE
GPU not showing both controllers

### DISCUSSION THREAD

**Discussion by IssacI19** [EvanMulawski/FanControl.CorsairLink#61] [General]
GPU not showing both controllers
![Screenshot 2023-06-30 123238](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/137911543/bd59045f-e6c8-4010-9331-d394159f5209)

![image](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/137911543/72386901-6fed-477c-9ce3-fb65d3547052)
In Fan Control under the Controls section shows up the 2 controllers. But under the speeds section of Fan Control, there's only one "Fan", and the second one is not showing

![image](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/137911543/4bd341c8-aefc-44e2-86f6-fb8d4966c91e)
The second fan controller seems to work but is not in sync with the other controller. For context "GPU Fan 1" is apparently the left fan and the middle fan. and "GPU Fan 2" is the right fan. Both controllers are linked to the "Fan 1" speed in Fan Control, but the first 2 fans are the ones always spinning first and the third fan spins later. Im assuming the third fan is not using Fan Control at all and using the default fan curve that it came with. Is there any way to fix this and have both Fan speeds RPM sensors show up?

**Comment by EvanMulawski** [comment]
Off-topic for this repo. Already posted here: https://github.com/Rem0o/FanControl.Releases/discussions/1870
